src/components/annotator/AppellationCreator.vue

```diff
@@ -159,17 +241,23 @@ export default class AppellationCreator extends Vue {
 			}
 		}
 	}
+
 	private cancel() {
 		this.$store.commit('setAnnotatorHighlightedText', null);
 		this.$store.commit('setAnnotatorSelectedConcept', null);
 		this.$store.commit('setAnnotatorEditAppellationMode', null);
+		this.isDateAppellation = false;
+		this.year = '';
+		this.month = null;
+		this.day = '';
+		this.$store.commit('setAnnotatorisDateAppellation', false);
 	}
 
 	private createOrUpdate() {
 		this.creating = true;
 		this.createError = false;
 		const highlighted = this.$store.getters.getAnnotatorHighlightedText;
-		const payload = {
+		const payload: any = {
```

Comment:
Should this just have its own type?

Reply:
Since the fields inside the payload will be different based on the type of appellation i.e date, datestring, concept, payload will need to contain all of the fields?